src/main/java/school/lemon/changerequest/java/introduction/hw1/Task10.java

```diff
@@ -3,7 +3,24 @@
 
 public class Task10 {
 
-    public static boolean containDigitTwo(int n) {
-        return false;
+    public static boolean containDigitTwo(int n)
+    {
+        int j=2;
+        for (int i = 1; i < j; i++)
+        {
+            float Num = n /10f;
```

Comment:
Variables and method names starts with lowercase letters. It's just a good practice in Java world.